yaml/simulator.yaml

```diff
@@ -29,7 +29,7 @@
     cmd: >
       <out>/vcs_simv +vcs+lic+wait <sim_opts> +ntb_random_seed=<seed> <cov_opts>
     cov_opts: >
-      -cm_dir <out>/test.vdb -cm_log /dev/null -cm_name test_<seed>
+      -cm_dir <out>/test.vdb -cm_log /dev/null -cm_name test_<test_id>
```

Comment:
Do you need to keep the seed? If the test is run multiple times with different seeds, this will end up overriding the same coverage directory.

Reply:
Good point, I've appended the seed to this tag as well